flexget/plugins/input/anidb_feed.py

```diff
@@ -0,0 +1,485 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+import logging
+import re
+from xml.dom import minidom
+
+import feedparser
+from time import mktime
+from dateutil.parser import parse as dateutil_parse
+from dateutil.tz import tz
+from datetime import datetime
+
+from flexget.config_schema import one_or_more
+
```

Comment:
I stopped using it a few months ago because it started doing weird stuff like this.